docs/Language.md

```diff
@@ -10,8 +10,18 @@ The following guidelines for writing copy are intended to create a consistent st
     * **Don't**: Diary Notes
 
 * **Use a friendly tone and avoid niceties** to appear more conversational
-    * **Do**:  Nice! You've completed all of your outstanding tasks.
+    * **Do**: Nice! You've completed all of your outstanding tasks.
     * **Don't**: Task completed successfully. All outstanding tasks have been completed.
+    * **Do**: All done. The job end date has been finalized.
+    * **Don't**: Job end date has successfully been finalized.
+    * **Do**: Exciting! Your new starter has been created.
+    * **Don't**: Person successfully created.
+    * **Do**: All set. John Wick's job record is ready to go.
+    * **Don't**: Job successfully created.
+    * **Do**: 8 forms have been scheduled. We'll automatically email the recipients when it's time.
+    * **Don't**: Successfully scheduled 8 forms.
+    * **Do**: Goal Template has been enabled.
+    * **Don't**: Goal Template has been enabled successfully.
```

Comment:
Any specifications for small things like adding phone number or email address or updating or editing similar scenarios.
For now I have used: You've removed/updated/edited your Email Address.

Reply:
The user doesn't need to be told what they've done because they're the ones who did it. It's probably only really appropriate in places where we're also using `Your` like `Your Outstanding Tasks`.

Reply:
I'll add more examples and clarification